pkg/test/resourcefixture/testdata/basic/bigquerydatatransfer/v1beta1/bigquerydatatransferconfig/bigquerydatatransferconfig-scheduledquery/_http.log

```diff
@@ -283,6 +129,7 @@ X-Xss-Protection: 0
     "write_disposition": "WRITE_APPEND"
   },
   "schedule": "first sunday of quarter 00:00",
+  "state": 2,
```

Comment:
Just curious, is this added `state` the result of not setting `destinationDatasetId`?

Reply:
Good catch. I am actually not sure. The "state" is hardcoded as PENDING (2) in the mock. Not sure why it was missing in the previous http log.

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/e06b44c45ddba0c776369bef1c82b4d78fd4814d/mockgcp/mockbigquerydatatransfer/datatransfer.go#L110